syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -387,13 +385,8 @@ def zero(self, shape=None):
 
         if shape is None or len(shape) == 0:
             shape = self.shape if self.shape else [1]
-        zero = torch.zeros(*shape, dtype=self.torch_dtype).share(
-            *self.locations,
-            field=self.field,
-            dtype=self.dtype,
-            crypto_provider=self.crypto_provider,
-            **no_wrap,
-        )
+        zero = torch.zeros(*shape, dtype=self.torch_dtype)
+        zero = self._to_shared_tensor(zero, *self.locations)
```

Comment:
I would prefer the previous option which is more readable .share(**self.get_class_attributes(), **no_wrap) which is more readable!